app/src/main/res/values/strings.xml

```diff
@@ -56,70 +56,70 @@
     <string name="butn_close">Close</string>
     <string name="text_deviceModelInfo">Device name</string>
     <string name="butn_clearQueue">Clear the queue</string>
-    <string name="text_pendingTransfersOfDevice">Pending Queue - %s</string>
+    <string name="text_pendingTransfersOfDevice">Pending queue - %s</string>
     <string name="butn_pendingTransfers">Queue</string>
     <string name="butn_shareWithTrebleshot">Share with TrebleShot</string>
     <string name="mesg_somethingWentWrong">Something went wrong</string>
     <string name="text_findDevicesHint">To find nearby devices tap the search icon</string>
     <string name="text_autoScanDevices">Scan devices automatically</string>
-    <string name="text_autoScanDevicesSummary">Scans devices when it is necessary</string>
+    <string name="text_autoScanDevicesSummary">Scans devices when necessary</string>
     <string name="mesg_stillScanning">Still scanning</string>
     <string name="text_advanced">Advanced</string>
     <string name="text_storagePath">Storage path</string>
     <string name="text_storagePathSummary">New files will be saved to this folder</string>
     <string name="mesg_pathUnavailable">%s currently unavailable</string>
-    <string name="butn_showOnFileManager">Show on file manager</string>
+    <string name="butn_showOnFileManager">Show in file manager</string>
     <string name="text_chooseFileManager">Choose file manager</string>
-    <string name="text_ipAddress">IP Address</string>
-    <string name="text_allowToAccess">Allow to access</string>
+    <string name="text_ipAddress">IP address</string>
+    <string name="text_allowToAccess">Allow access to</string>
```

Comment:
I think in this context, previous one was better.
![screenshot_20180114-180234](https://user-images.githubusercontent.com/10258162/34917296-b4d8f4ae-f955-11e7-93b5-4fc5a675a877.png)